ui/tutorial-renderer/src/main/res/layout/tutorial_tips_start.xml

```diff
@@ -5,69 +5,19 @@
         <variable name="callbacks" type="org.cru.godtools.tutorial.TutorialCallbacks" />
     </data>
 
-    <androidx.constraintlayout.widget.ConstraintLayout
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:clipToPadding="false"
-        android:paddingTop="@dimen/tutorial_page_inset_top"
-        android:paddingBottom="@dimen/tutorial_page_inset_bottom">
 
-        <include layout="@layout/tutorial_include_margins" />
 
-        <include layout="@layout/tutorial_tips_include_positioning" />
+
         <androidx.compose.ui.platform.ComposeView
             android:id="@+id/compose"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             app:layout_constraintStart_toStartOf="@id/marginStart"
             app:layout_constraintEnd_toEndOf="@id/marginEnd"
-            app:layout_constraintTop_toTopOf="parent" />
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintBottom_toTopOf="@id/action_tips_finish"/>
```

Comment:
we can remove all the `app:layout_constraint*` attributes now that this view isn't in a `ConstraintLayout`